app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```

Comment:
- [ ] It would be better if comment is also created on behalf of the currently loggedIn user as in the requirements
![Screenshot from 2023-05-26 00-28-11](https://github.com/fithamlak/blog-app/assets/46388113/360f69c5-c525-42da-98e3-533fd93ca1b0)

Reply:
if I am not wrong it is the same style as creating post it takes from comment_params